src/content/docs/data-apis/ingest-apis/event-api/incident-event-rest-api.mdx

```diff
@@ -0,0 +1,312 @@
+
+The incident API is built on our [event API](/docs/data-apis/ingest-apis/introduction-event-api). 
+
+The incident event API is an asynchronous endpoint. This means you can send a large volume of POSTS, reliably, with low-response latency.
+
+## Using the API: an overview [#overview] 
+
+Here's the general process for sending custom incident events to New Relic via the incident event API:
+
+1. Generate a [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key) for the account you want to report data to.
+2. Before you get started creating custom attributes, review the [event API's limits and restricted characters](/docs/data-apis/ingest-apis/introduction-event-api/#limits).
+3. Generate the JSON for your incident event that follows our [JSON format guidelines](/docs/data-apis/ingest-apis/introduction-event-api#instrument).
+4. Submit a compressed JSON payload (for example, gzip or deflate) to the HTTPS endpoint using curl in a POST request.
+
+Once you've started sending incident event data to New Relic, we recommend that you set up [NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) to get notifications about parsing errors.
+
+## Example JSON [#example-json]
+
+Your JSON payload may look something like this example.
+
+```json
+[
+  {
+    "eventType": "NrAiIncidentExternal",
+    "title": "Test",
+    "description": "The latency is above threshold of 500000 MS",
+    "state": "trigger",
+    "source": "luna",
+    "entityName": "testEntity",
+    "entity.guid": "testEntity123",
+    "aggregationTag.serviceId": 5,
+    "aggregationTag.environment": "testing",
+    "aggregationTag.errorId": 10543,
+    "tag.stackTrace": "some stack trace...",
+    "version": 1  
+  }
+]
+```
+
+The `eventType` field must use `NrAiIncidentExternal`.
+
+## Send your incident event from the command line [#cli]
+
+Use curl to send your JSON payload.
+
+```bash
+gzip -c example_incidents.json | curl --data-binary @- -X POST -H "Content-Type: application/json" -H "X-Insert-Key: <var>YOUR_INSERT_KEY</var>" -H "Content-Encoding: gzip" 
+https://staging-insights-collector.newrelic.com/v1/accounts/<var>YOUR_ACCOUNT</var>/events
+```
+
+```bash
+Response 
+{"success":true, "uuid":"<var>YOUR_NEW_RELIC_ACCOUNT_ID</var>"}%
+```
+
+Once you've successfully sent your incident event data, see it in the UI or via [NRDB, the New Relic database](/docs/data-apis/get-started/nrdb-horsepower-under-hood).
+
+### Example NRDB query for incident events [#nrdb-example]
+
+You can query the New Relic database with [Nerdgraph, our GraphQL API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/).
+
+```json
+{
+  actor {
+    account(id: <var>YOUR_ACCOUNT_ID</var>) {
+      nrql(query: "SELECT * FROM NrAiInternalIncident since 3 days ago") {
+        results
+      }
+    }
+  }
+}
+
+
+         "results": [
```

Comment:
i think wrapping `"results": [...]` in curlies `{}` may make the scary red text go away

Reply:
oh and it's missing the closing `]`